ci/appveyor-bootstrap.py

```diff
@@ -20,14 +20,12 @@
 GET_PIP_URL = "https://bootstrap.pypa.io/get-pip.py"
 GET_PIP_PATH = "C:\get-pip.py"
 URLS = {
-    ("2.7", "64"): BASE_URL + "2.7.14/python-2.7.14.amd64.msi",
-    ("2.7", "32"): BASE_URL + "2.7.14/python-2.7.14.msi",
-    ("3.4", "64"): BASE_URL + "3.4.8/python-3.4.8.amd64.msi",
-    ("3.4", "32"): BASE_URL + "3.4.8/python-3.4.8.msi",
-    ("3.5", "64"): BASE_URL + "3.5.5/python-3.5.5-amd64.exe",
-    ("3.5", "32"): BASE_URL + "3.5.5/python-3.5.5.exe",
```

Comment:
why no support for 3.5?

Reply:
Because it's old enough that the CI's were having issues with it.